moveit_setup_assistant/src/widgets/planning_groups_widget.cpp

```diff
@@ -1045,7 +1045,7 @@ void PlanningGroupsWidget::saveSubgroupsScreen()
 }
 
 // ******************************************************************************************
-// Call when groups edit sceen is done and needs to be saved
```

Comment:
This whole PR would be much smaller if you separated the spelling fixes into a separate PR.  Please do that.